snooty/parser.py

```diff
@@ -1002,6 +1007,25 @@ def validate_tabs_children(self, node: n.Directive) -> None:
         node.children = new_children
         return
 
+    def validate_icon_role(self, node: docutils.nodes.Node) -> None:
+        """
+        Validate target for icon role
+        Checks for included icon file in root path
+        """
+        if not ICON_SET:
+            return
+        icon_name = node["target"]
+        possible_name_set = {
```

Comment:
non-blocking nitpick: We can name this something more contextual like `icon_name_formats` - since we're not exporting the var for usage, I'd be comfortable not hinting the type in the var name
```suggestion
        icon_name_formats = {
```

Reply:
good point. redundant without exposure. updated!